packs/drupal.js

```diff
@@ -0,0 +1,44 @@
+  /** Additional description of a Lighthouse audit that tells the user how they can improve site loading performance by reducing the total bytes delivered by their page in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  total_byte_weight: 'Consider using [Responsive Image Styles](https://www.drupal.org/docs/8/mobile-guide/responsive-images-in-drupal-8) to reduce the size of images loaded in your page. If you are using Views to show multiple content items in a page, consider implementing pagination to limit the number of content items shown on a given page.',
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by reducing the amount of render blocking resources present on their page, in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  render_blocking_resources: 'Consider using the [Advanced CSS/JS Aggregation](https://www.drupal.org/project/advagg) module to inline critical CSS or potentially load assets asynchronously via JavaScript. Beware that optimizations provided by this module may break your site, so you will likely need to make code changes.',
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by minifying their CSS files in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  unminified_css: 'Ensure you have enabled "Aggregate CSS files" in the "Administration » Configuration » Development" page. You can also use the [Advanced CSS/JS Aggregation](https://www.drupal.org/project/advagg) module to speed up your site by concatenating, minifying, and compressing your styles.',
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by minifying their Javascript files in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  unminified_javascript: 'Ensure you have enabled "Aggregate JavaScript files" in the "Administration » Configuration » Development" page. You can also use the [Advanced CSS/JS Aggregation](https://www.drupal.org/project/advagg) module to speed up your site by concatenating, minifying, and compressing your JavaScript assets.',
```

Comment:
Since we mention `Advanced CSS/JS Aggregation` multiple times, I'm okay with it if it happens to be the canonical module for minifying/aggregating. Will leave that up to you to decide :)

Reply:
Same thing here, Advanced CSS/JS Aggregation is _almost_ the canonical module that can offer more granularity and advanced aggregation options - hence the name.

Nevertheless, I have rewritten the recommendation to open up possibilities of other modules.

It now reads as:
**unminified_css**
> Ensure you have enabled "Aggregate CSS files" in the "Administration » Configuration » Development" page. You can also configure more advanced aggregation options through [additional modules](https://www.drupal.org/project/project_module?f%5B0%5D=&f%5B1%5D=&f%5B2%5D=im_vid_3%3A123&f%5B3%5D=&f%5B4%5D=sm_field_project_type%3Afull&f%5B5%5D=&f%5B6%5D=&text=css+aggregation&solrsort=iss_project_release_usage+desc&op=Search) to speed up your site by concatenating, minifying, and compressing your CSS styles.


**unminified_javascript**
> Ensure you have enabled "Aggregate JavaScript files" in the "Administration » Configuration » Development" page. You can also configure more advanced aggregation options through [additional modules](https://www.drupal.org/project/project_module?f%5B0%5D=&f%5B1%5D=&f%5B2%5D=im_vid_3%3A123&f%5B3%5D=&f%5B4%5D=sm_field_project_type%3Afull&f%5B5%5D=&f%5B6%5D=&text=javascript+aggregation&solrsort=iss_project_release_usage+desc&op=Search) to speed up your site by concatenating, minifying, and compressing your JavaScript assets.